doc/Programming-guide/Using-Kaa-endpoint-SDKs/index.md

```diff
@@ -11,14 +11,14 @@ sort_idx: 40
 * TOC
 {:toc}
 
-Developing an IoT solution we all the time facing the same routine: creating network communication stack, log delivery functionality, event exchange between endpoints, etc.
-The Kaa platform already provides all of this functionality, and you can get it out of the box using Kaa endpoint SDK.
+Developing a [Kaa instance]({{root_url}}Glossary/#kaa-instance-kaa-deployment) requires following the same routine: creating network communication stack, log delivery functionality, event exchange between endpoints, etc.
```

Comment:
I think you said the same but in other words, which means your understanding is correct anyway :) All I did there is I changed 'IoT solution' to 'Kaa instance' to be more precise why I mention that 'routine' -- I don't think it's safe to assume it's the same routine for _every_ IoT solution out there.